common.go

```diff
@@ -77,6 +77,7 @@ type dbTerm struct {
 	Expression string
 	Reading    string
 	Tags       []string
```

Comment:
At the point where we have `termTags`, it would probably make sense to rename `tags` to be be something that more specifically reflects what they are tagging.